.travis.yml

```diff
@@ -5,7 +5,7 @@ jdk:
 - oraclejdk7
 - openjdk7
 script:
-- if java -version 2>&1 | grep -q 'java version "1.8.'; then ./gradlew runFunctionalTest; else true; fi
+- if java -version 2>&1 | grep -q 'java version "1.8.'; then ./gradlew check; ./gradlew runFunctionalTest; else true; fi
```

Comment:
By default travis does `check` for gradle projects. https://docs.travis-ci.com/user/languages/java/#Projects-Using-Gradle.  Looks like `script` overrides this.  If so, you would need to run `./gradlew check` for all java versions not only for `java version 1.8`